edx_exams/apps/api/v1/views.py

```diff
@@ -426,19 +426,19 @@ def get(self, request):
         """
         user = request.user
         latest_attempt = get_latest_attempt_for_user(user.id)
+        serialized_attempt = StudentAttemptSerializer(latest_attempt)
 
         # if there is an active attempt in this service, return it
         if latest_attempt is not None:
             latest_attempt = check_if_exam_timed_out(latest_attempt)
-            serialized_attempt = StudentAttemptSerializer(latest_attempt)
-
             if latest_attempt.status in (ExamAttemptStatus.started, ExamAttemptStatus.ready_to_submit):
                 return Response(status=status.HTTP_200_OK, data=serialized_attempt.data)
 
         # if edx-proctoring has an active attempt, return it
         latest_attempt_legacy, response_status = get_active_exam_attempt(user.lms_user_id)
         if latest_attempt_legacy is not None and response_status == status.HTTP_200_OK:
```

Comment:
looks like this could just be merged with the new condition so we only have a single if statement here. That would make the line long but you could refactor to this format:
```suggestion
        if (
            latest_attempt_legacy is not None
            and latest_attempt_legacy != {}
            and response_status == status.HTTP_200_OK
        ):
```